test/testapps/ClassLibrary/ClassLibrary.csproj

```diff
@@ -1,25 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
   
   <PropertyGroup>
-    <_RazorMSBuildRoot>$(SolutionRoot)src\Microsoft.AspNetCore.Razor.Design\bin\$(Configuration)\</_RazorMSBuildRoot>
-    <RazorSdkBuildTasksDirectoryRoot>$(RazorSdkProjectDirectory)bin\$(Configuration)\</RazorSdkBuildTasksDirectoryRoot>
+    <RazorSdkDirectoryRoot>$(RazorSdkProjectDirectory)bin\$(Configuration)\sdk-output\</RazorSdkDirectoryRoot>
   </PropertyGroup>
 
-  <Import Project="$(SolutionRoot)src\Microsoft.AspNetCore.Razor.Design\build\netstandard2.0\Microsoft.AspNetCore.Razor.Design.props" />
-
-  <PropertyGroup>
-    <!-- Override for the MVC extension -->
-    <_MvcExtensionAssemblyPath>$(SolutionRoot)src\Microsoft.AspNetCore.Mvc.Razor.Extensions\bin\$(Configuration)\netstandard2.0\Microsoft.AspNetCore.Mvc.Razor.Extensions.dll</_MvcExtensionAssemblyPath>
-  </PropertyGroup>
-  <Import Project="$(SolutionRoot)src\Microsoft.AspNetCore.Mvc.Razor.Extensions\build\netstandard2.0\Microsoft.AspNetCore.Mvc.Razor.Extensions.props" />
-
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
 
     <Copyright>© Microsoft</Copyright>
     <Product>Razor Test</Product>
     <Company>Microsoft</Company>
     <Description>ClassLibrary Description</Description>
+
+    <_EnableAllInclusiveRazorSdk>true</_EnableAllInclusiveRazorSdk>
```

Comment:
hmmmm - I think I want to understand this

Reply:
These are *technically* 3.0 targeting RCLs (without references to Razor.Design). Consequently we need for a way to tell the SDK to use the targets it ships. Essentially this - https://github.com/aspnet/Razor/issues/2676. There's an existing 2.1 RCL testapp that does not require this